config.py

```diff
@@ -373,10 +373,12 @@ class configuration:
                               ('DOUBLE', None),
                               ('NVCC', None),
                               ('CCACHE_CC', 'nvcc')]
-  # define OPENMP
+
+  # define OPENMP and SWIG (for log output)
   for compiler in macros:
     for precision in macros[compiler]:
       macros[compiler][precision].append(('OPENMP', None))
+      macros[compiler][precision].append(('__SWIG__', None))
```

Comment:
Any reason you use ``__SWIG__`` here rather than ``SWIG``? If so, perhaps all of our macros in "config.py" should be renamed with the double underscores.